<?xml version="1.0"?>
<launch>

    <arg name="rviz" default="true" />
    <arg name="project" default="multi_sensor_mapping"/>

    <!-- Lidar IMU Mapper -->
    <node pkg="$(arg project)" type="low_cost_mapping_node" name="low_cost_mapping_node" output="screen" respawn="false">
    <!-- <node pkg="$(arg project)" type="low_cost_mapping_node" name="low_cost_mapping_node" output="screen" respawn="false" launch-prefix="gdb -ex run &#45;&#45;args"> -->
        <param name="config_name" value="S3E"/>
        <param name="map_cache_path" value="/home/hkw/dataset/backpack/shihushan_test1"/>
    </node>

        <!--- Rviz -->
    <group if="$(arg rviz)">
      <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find multi_sensor_mapping)/rviz/lidar_mapper.rviz" />
    </group>


</launch>
